spec/models/classification_spec.rb

```diff
@@ -571,6 +571,41 @@
     end
   end
 
+  describe '#save' do
+    let(:new_name) { "new_tag_name" }
+    let(:category) { FactoryBot.create(:classification, :name => "category") }
+
+    context "editing existing classification" do
+      let(:classification) { FactoryBot.create(:classification_tag, :parent => category, :name => "some_tag_name") }
+      it "doesn't assign new tag " do
+        tag = classification.tag
+        classification.update_attributes!(:name => new_name)
+        classification.reload
+        expect(tag.id).to eq classification.tag.id
+        expect(classification.name).to eq(new_name)
```

Comment:
👍 clear explanation what is difference between classification name and tag name

Reply:
I always find these confusing.

```ruby
tag_name = "/managed/category/classification"
classification.name = "classification"
```